packages/kbn-es/src/integration_tests/__fixtures__/es_bin.js

```diff
@@ -31,11 +31,36 @@ const { ES_KEY_PATH, ES_CERT_PATH } = require('@kbn/dev-utils');
     },
     (req, res) => {
       const url = new URL(req.url, serverUrl);
-      const send = (code, body) => {
-        res.writeHead(code, { 'content-type': 'application/json' });
+      const send = (code, body, headers = {}) => {
+        res.writeHead(code, { 'content-type': 'application/json', ...headers });
         res.end(JSON.stringify(body));
       };
 
+      // ES client's Product check request: it checks some fields in the body and the header
+      if (url.pathname === '/') {
+        return send(
+          200,
+          {
+            name: 'es-bin',
```

Comment:
why didn't this test contain this object before?

Reply:
`es_bin` was not covering the `GET /` requests. It'd just returned 404 because of the default use case. This means, that the client would never pass the Product Check and all the `kbn-es` tests would fail.

We can either disable the product check for these tests, or handle the specific request to pass it and unblock the client from that state.